x-pack/plugins/security_solution/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -5,16 +5,199 @@
+        },
+      },
+      {
+        field: 'created_by',
+        name: 'Created by',
+        truncateText: true,
+        render: (name: string) => {
+          return (
+            <EuiFlexGroup responsive={false} gutterSize={'xs'} alignItems="center">
+              <EuiFlexItem grow={false}>
+                <EuiAvatar name={name} data-test-subj={'created-by-avatar'} size="s" />
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <TextValueDisplay>{name}</TextValueDisplay>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          );
+        },
+      },
+      {
+        field: 'created_at',
+        name: createdAtColumnName,
+        truncateText: true,
+        render: (date: string) => {
+          return (
+            <FormattedDate
+              fieldName={createdAtColumnName}
+              value={date}
+              className="eui-textTruncate"
+            />
+          );
+        },
+      },
+      {
+        field: 'updated_by',
+        name: 'Last updated by',
+        truncateText: true,
+        render: (name: string) => {
+          return (
+            <EuiFlexGroup responsive={false} gutterSize={'xs'} alignItems="center">
+              <EuiFlexItem grow={false}>
+                <EuiAvatar name={name} data-test-subj={'updated-by-avatar'} size="s" />
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <TextValueDisplay>{name}</TextValueDisplay>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          );
+        },
+      },
+      {
+        field: 'updated_at',
+        name: updatedAtColumnName,
+        truncateText: true,
+        render: (date: string) => {
+          return (
+            <FormattedDate
+              fieldName={updatedAtColumnName}
+              value={date}
+              className="eui-textTruncate"
+            />
+          );
+        },
+      },
+      { field: '-', name: 'Endpoints' },
+      { field: '-', name: 'Actions' },
+    ];
+  }, []);
+
+  const handleTableOnChange = useCallback(
+    ({ page }: CriteriaWithPagination<PolicyData>) => {
+      setPagination({
+        currentPage: page.index + 1,
+        pageSize: page.size,
+      });
+    },
+    [setPagination]
+  );
+
+  const tablePagination = useMemo(() => {
+    return {
+      pageIndex: pagination.currentPage - 1,
+      pageSize: pagination.pageSize,
+      totalItemCount,
+      pageSizeOptions,
+    };
+  }, [totalItemCount, pageSizeOptions, pagination.currentPage, pagination.pageSize]);
+
   return (
     <AdministrationListPage
       data-test-subj="policyListPage"
-      title="Policy List"
-      subtitle="list of all the policies"
-    />
+      title={i18n.translate('xpack.securitySolution.policy.list.title', {
+        defaultMessage: 'Policy List',
+      })}
+      subtitle={i18n.translate('xpack.securitySolution.policy.list.subtitle', {
+        defaultMessage: 'List of all the policies',
```

Comment:
is this the actual text for the page? or are you still waiting for the actual text to put in here

Reply:
still waiting for the actual description @caitlinbetz

Reply:
Ok. If you don't get it in time for this PR, can you please open an issue to track so we don't forget?